src/clj/game/cards/hardware.clj

```diff
@@ -796,6 +796,30 @@
                           :effect (effect (update! (dissoc card :Omnidrive-prog))
                                           (use-mu (:memoryunits target)))}}}
 
+   "Patchwork"
+   (letfn [(patchwork-discount [cost-type bonus-fn]
+             {:async true
+              :req (req (and (not (get-in @state [:per-turn (:cid card)])) ; manual once-per-turn check
+                             (= "Runner" (:side target))
+                             ;; The card being installed/played is still in the hand; we therefore need at least 2
+                             ;; cards for this effect to make sense.
+                             (< 1 (count (:hand runner)))))
```

Comment:
If I install something via SMC or play an event via Same Old Thing with one card in hand, Patchwork *should* still trigger

Reply:
(Also, Patchwork won't give +1 Memory atm)